### PR TITLE
Fix examples in docs and update for recent work.

### LINKED ISSUE #105: When running examples from documentation, requests hang due to http
Documentation says you can use http, but the server requires https.

### PR DESCRIPTION
**Pull request by yupinghu**
Closes #105.

### CODE REVIEW

**Comment by ericvw** [comment]
lgtm

File: doc/http-api-guide.md [hunk 1]
```diff
@@ -5,21 +5,22 @@ The [Bloomberg Open API] provides access to market data. The
...
-Reference Data Service (i.e. //blp/refdata) and the API Field Service
-(//blp/apiflds).
-
 This document provides an overview of the [HTTP API]: the URLs to use and the
-formatting for the HTTP requests and responses. We assume, for the purposes of
-this doc, that the server at http://http-api.openbloomberg.com/ is running an
-instance of this API.
+formatting for the HTTP requests and responses. Bloomberg operates a server at
+https://http-api.openbloomberg.com/ that runs an instance of this API for demo
+and other purposes (e.g. hackathons). All examples in this document as well as
```
Comment (ericvw) [inline doc/http-api-guide.md]: ...document, as well as elsewhere i this project,

Add `,`

File: doc/http-api-guide.md [hunk 2]
```diff
@@ -5,21 +5,22 @@ The [Bloomberg Open API] provides access to market data. The
...
+elsewhere in this project are written to work with this server, e.g. they use
```
Comment (ericvw) [inline doc/http-api-guide.md]: Wrap the `(e.g.,)`

File: doc/http-api-guide.md [hunk 3]
```diff
@@ -5,21 +5,22 @@ The [Bloomberg Open API] provides access to market data. The
...
+client certificates for authentication/identity.
 
 This document also includes basic information about some of the underlying
 [Open API] operations. For more details on the [Open API], refer to the
 [BLPAPI Developer's Guide]. Where appropriate, this document will refer to
 specific sections of the [Developer's Guide].
 
+The [HTTP API] is currently in development and future versions may add or
+change functionality. For example, additional authentication options may become
```
Comment (ericvw) [inline doc/http-api-guide.md:21]: See!  The `,` is here!

**Comment by ericvw** [comment]
Withdrawing `lgtm` based on looking more in detail :D

**Comment by yupinghu** [comment]
Updated for review feedback.

File: doc/http-api-guide.md [hunk 4]
```diff
@@ -5,21 +5,22 @@ The [Bloomberg Open API] provides access to market data. The
...
-Reference Data Service (i.e. //blp/refdata) and the API Field Service
-(//blp/apiflds).
-
 This document provides an overview of the [HTTP API]: the URLs to use and the
-formatting for the HTTP requests and responses. We assume, for the purposes of
-this doc, that the server at http://http-api.openbloomberg.com/ is running an
-instance of this API.
+formatting for the HTTP requests and responses. Bloomberg operates a server at
+https://http-api.openbloomberg.com/ that runs an instance of this API for demo
+and other purposes (e.g. hackathons). All examples in this document, as well as
```
Comment (ericlu88) [inline doc/http-api-guide.md]: one more ','

**Comment by ericlu88** [comment]
LGTM.

**Comment by yupinghu** [comment]
Added some commas. :)

**Comment by ericvw** [comment]
lgtm - please squash

**Comment by yupinghu** [comment]
Squashed

**Pull request merged**